### ISSUE TITLE
`$ rails g mailer ` not generating `layouts/mailer` etc.

### ISSUE THREAD

**Issue by mbrand12** [slim-template/slim-rails#95]
When I run `$ rails generate mailer UserMailer account_activation password_reset`

on a rails app with no slim-rails:

```
      create  app/mailers/user_mailer.rb
      create  app/mailers/application_mailer.rb
      invoke  erb
      create    app/views/user_mailer
      create    app/views/layouts/mailer.text.erb
      create    app/views/layouts/mailer.html.erb
      create    app/views/user_mailer/account_activation.text.erb
      create    app/views/user_mailer/account_activation.html.erb
      create    app/views/user_mailer/password_reset.text.erb
      create    app/views/user_mailer/password_reset.html.erb
      invoke  test_unit
      create    test/mailers/user_mailer_test.rb
      create    test/mailers/previews/user_mailer_preview.rb
```

with slim-rails:

```
      create  app/mailers/user_mailer.rb
      create  app/mailers/application_mailer.rb
      invoke  slim
      create    app/views/user_mailer
      create    app/views/user_mailer/account_activation.text.slim
      create    app/views/user_mailer/password_reset.text.slim
      invoke  test_unit
      create    test/mailers/user_mailer_test.rb
      create    test/mailers/previews/user_mailer_preview.rb
```

Missing are:

```
      create    app/views/layouts/mailer.text.slim
      create    app/views/layouts/mailer.html.slim
      create    app/views/user_mailer/account_activation.html.slim
      create    app/views/user_mailer/password_reset.html.slim
```

Any specific reason why, or a setting that I may have overlooked?

The only setting I have is:
```
  Slim::Engine.set_options pretty: true, sort_attrs: false

```

in `environments/development.rb`

**Comment by Sathors** [comment]
I can confirm the problem.
Nobody to merge the pull request?

**Comment by Salzig** [comment]
merged and released as 3.1.0.
